src/components/WellcomeCollectionBanner/WellcomeCollectionBanner.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { CookiesProvider, useCookies } from 'react-cookie';
+
+import Button from 'Button/Button';
+import Icon from 'Icon/Icon';
+
+export const WellcomeCollectionLink = () => {
+  const IS_WELLCOME_COLLECTION_LINK_SEEN = 'isWellcomeCollectionLinkSeen';
```

Comment:
Presuming the all caps are convention for cookie names/vars?

Reply:
Yeah, all caps is a convention for declaring constants (not necessarily the same as `const`)

https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/